### PR TITLE
Homework12: Anastasiya Solodukho

### PR DESCRIPTION
**Pull request by AnastasiyaSo**
Added homework12

### CODE REVIEW

File: homework/homework12/homework12_1.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+class CardsDeck:
+    """Creating cards deck class"""
+    def __init__(self, deck_number_list, deck_mast_list):
+        self.cards_list = []
+        for i in deck_mast_list:
+            for n in deck_number_list:
+                self.cards_list.append(Card(i, n))
+        self.cards_list.append(Card("Joker", None))
+        self.cards_list.append(Card("Joker", None))
```
Comment (Stml89) [inline homework/homework12/homework12_1.py]:
```suggestion
        self.cards_list.append(Card("Joker", "Red"))
        self.cards_list.append(Card("Joker", "Black"))
```

File: homework/homework12/homework12_1.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        self.mast = mast
+        self.number = number
+
+
+class CardsDeck:
+    """Creating cards deck class"""
+    def __init__(self, deck_number_list, deck_mast_list):
+        self.cards_list = []
+        for i in deck_mast_list:
+            for n in deck_number_list:
```
Comment (Stml89) [inline homework/homework12/homework12_1.py]:
```suggestion
        for i in Card.mast_list:
            for n in Card.number_list:
```

File: homework/homework12/homework12_2.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+usd = Currency("USD", 3.26)
+byn = Currency("BYN", 1)
+
+
+class Bank:
+    """Creating banks class"""
+    def __init__(self, name):
+        self.name = name
+
+    def exchange_currency(self, currency_from, person_amount,
```
Comment (Stml89) [inline homework/homework12/homework12_2.py:32]: move it to the `currency` class

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework/homework12/homework12_2.py (hunk 3 above)
Comment (AnastasiyaSo) [reply homework/homework12/homework12_2.py:32]: I don't understand why. In HW we should expand Banks methods

**Comment by AnastasiyaSo** [comment]
@Stml89, I added deposit method from HW 11. Please, check

**Review by Stml89 — APPROVED** [review]

**Pull request merged**